client/package.json

```diff
@@ -4,17 +4,26 @@
   "description": "This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).",
   "main": "index.js",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.25",
+    "@fortawesome/free-regular-svg-icons": "^5.11.2",
+    "@fortawesome/free-solid-svg-icons": "^5.11.2",
+    "@fortawesome/react-fontawesome": "^0.1.7",
+    "antd": "^3.24.3",
     "axios": "^0.19.0",
+    "node-sass": "^4.13.0",
     "react": "^16.10.2",
+    "react-calendar": "^2.19.2",
     "react-dom": "^16.10.2",
+    "react-dropdown": "^1.6.4",
     "react-redux": "^7.1.1",
     "react-router-dom": "^5.1.2",
     "react-scripts": "^3.2.0",
     "redux": "^4.0.4",
     "redux-logger": "^3.0.6",
     "redux-thunk": "^2.3.0",
     "semantic-ui-css": "^2.4.1",
-    "semantic-ui-react": "^0.88.1"
+    "semantic-ui-react": "^0.88.1",
+    "styled-components": "^4.4.1"
```

Comment:
I saw a few examples of styles in a sass file and styles in the component file. I'd encourage y'all to decide on one approach and standardize on that. 😄